bosh-setup/ritaazuredeploy.parameters.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "value": "ritacfxregiondev"
+    },
+    "adminUsername": {
+      "value": "ritacfxregiondevadmin"
+    },
+    "sshKeyData": {
+      "value": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQj6+nT3zw7ql+UbyaVz8ElfVyZKZZf30/4li3bR0vD4jSp7LfSzKkWAhuoVg/3G0VCaUcDc+5DBtNNQB0Rv9yM6xVV/hycIlu+/AP12+kN0qDAruJxoRsUKCiv1kNxLuSXnCBCQ0nm3RKtk+LlyLuQOS/j/6i6e+niAb3zcbGXxZ9fy/DorJSzHx/A1ho6E8SkJtsIEIe5LmeRaQuPpAuVIuYwHMrXD6mYGcKBXrEUZJmY6lstYhkD5owL3ujLq/m0Np36mlpEipk6GnF+6jpAcfhuT5g17Whc02937Qgin6sirTvmcrYhtA4H6y9dwdh+UrNHN/KR06NuvNLKav7 ritazhang@Ritas-MacBook-Pro.local"
+    },
+    "environment": {
+      "value": "AzureCloud"
+    },
+    "tenantID": {
+      "value": "72f988bf-86f1-41af-91ab-2d7cd011db47"
+    },
+    "clientID": {
+      "value": "dfdbb26c-199b-4647-9117-325756f01818"
+    },
+    "clientSecret": {
+      "value": "Tummybed8457b!"
+    },
+    "autoDeployBosh": {
+      "value": "disabled"
+    },
+    "primaryLocation": {
+        "value": "West US"
+    },
+    "secondaryLocation": {
+        "value": "East US"
+    },
+    "systemdomain": {
+        "value": "todomydomain.com"
+    },
+    "dbendpoint": {
+        "value": "dxmariadblb.northeurope.cloudapp.azure.com"
+    },
+    "nfsendpoint": {
+        "value": "todonfsendpoint.com"
+    }
+  }
+}
```

Comment:
@ritazh 

This file should not be uploaded. You client secret is exposed and you'd better change it.